docs/content/en/docs/yaml-crd-ref/evaluationdefinition.md

```diff
@@ -1,5 +1,132 @@
 ---
 title: KeptnEvaluationDefinition
-description: Define all workloads and checks associated with an application
-weight: 25
+description: Define an evaluation query
+weight: 20
 ---
+
+A `KeptnEvaluationDefinition` assigns target values
+to [KeptnMetric](metric.md) queries.
+These are used as part of evaluation tasks
+that can be run by the Keptn Lifecycle Toolkit
+as part of pre- and post-analysis phases of a workload or application.
+You can then create a [KeptnTaskDefinition](taskdefinition.md)
+that describes an action to take based on the results of the evaluation.
+
+For example, you could define a `KeptnMetric`
+that reports the number of CPUs in your cluster.
+A `KeptnEvaluationDefinition` could then define the value as `>4`,
+and define a task to be executed if that value is not met.
```

Comment:
not true, see previous comment

Reply:
Sorry for not being more verbose. We actually cannot define a task to be executed if the value is not met. Basically we just evaluate the data and say if it met or didn't met out evaluation target. If it didn't met it, we do not proceed to the next stage, if yes then we proceed.

Reply:
Thanks so much for the information -- I learned something.  I think I fixed the prose to be correct.  If so, we can merge this!